train.py

```diff
@@ -219,7 +315,14 @@ def run_training(args, train_data, val_data):
 
 def main(args):
     tokenizer = AutoTokenizer.from_pretrained(args.model_path, use_auth_token=True)
-    train_dataset, eval_dataset = create_datasets(tokenizer, args)
+    train_dataset, eval_dataset, eval_no_fim = create_datasets(tokenizer, args)
```

Comment:
`eval_no_fim` is not used (see comment above). We can add an option if the user wants to evaluate on both datasets when `fim_rate` is not zero. To do that, you can pass a `dict` as `eval_dataset` to the `Trainer` ([see](eval_no_fim))

Reply:
That was left over from testing. I don't see why we would want non-FIM eval so I have removed for now.